test/pytest/test_qkeras.py

```diff
@@ -272,6 +272,7 @@ def randX_1000_1():
         (quantized_relu(8, 4)),
         (quantized_relu(10)),
         (quantized_relu(10, 5)),
+        (quantized_relu(10, 5, negative_slope=0.25)),
```

Comment:
Aren't you already testing this in the other test?

Reply:
In order to create the test suite requested for the PR I added the line in the already present qkeras test, however due to problems with the environment requested for the pytests, some issues not concerning my fix came up. So, I have written a separate script just to show that the "new" layer passes an accuracy test. If the `test_qkeras.py` test passes when called in the automatic workflow, my other script can be removed.